.discourse-compatibility

```diff
@@ -1,3 +1,4 @@
+< 3.3.0.beta2-dev: 0a837d7c044798417b1a7389cc37a67859c25dbe
```

Comment:
Where did you get this SHA from?

Reply:
Ah good catch. I have no idea where that SHA came from but I have now updated the SHA to point to https://github.com/discourse/discourse-brand-header/commit/e64cfa531121f2cf152901f5b2a20512a5ee2c63